projects/batfish-client/src/main/java/org/batfish/client/Client.java

```diff
@@ -792,6 +799,10 @@ private boolean answerType(
 
    private boolean cat(String[] words)
          throws IOException, FileNotFoundException {
+      if (words.length == 1) {
+         _logger.errorf(words[0] + " requires additional arguments\n");
```

Comment:
Do I need to explicitly check the number of options and parameters? If so, how can I determine what is the exact number of args for options and parameters required for each command? For example, in Command.buildUsageMap():
      descs.put(ADD_BATFISH_OPTION, new Pair<>(
            "<option-key> <option-value>",
            "Additional options to pass to Batfish"));
It looks like ADD_BATFISH_OPTION command requires two inputs? They are options or parameters? When we actually use ADD_BATFISH_OPTION, we do things like : add-batfish-option haltonparseerror .  It actually accept a single argument?